ProjectGroupA/src/main/java/com/fundation/search/View/ActionSearch.java

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
Need to check with the team, to have a similar header :)
Paolo was using 
/**
 *  @(#)AssetFile.java Copyright (c) 2018 Jalasoft.